terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,66 @@
 provision_custom_search = False
 
 
+def content_security_policy() -> str:
+    def q(s: str) -> str:
+        return f"'{s}'"
+
+    def s(*args: str) -> str:
+        return ' '.join(args)
+
+    self = q('self')
+    none = q('none')
+    unsafe_inline = q('unsafe-inline')
+    unsafe_eval = q('unsafe-eval')
+    # Note that trailing slashes in host paths are significant. Paths ending
+    # with a slash match any path they are a prefix of. Paths that don't end
+    # with a slash only match the specified path exactly. Finally, host values
+    # without a path match any path from that host.
+    # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#hosts_values
```

Comment:
```suggestion
```

Same reason, there are so many subtleties to CSPs that documenting this one seems arbitrary.